internal/controller/spannerautoscaler_controller_test.go

```diff
@@ -410,3 +425,57 @@ var _ = Describe("Get and overwrite scaledown interval", func() {
 		Expect(got).To(Equal(want))
 	})
 })
+
+var _ = Describe("Get and overwrite scaleup interval", func() {
+	var testReconciler *SpannerAutoscalerReconciler
+	controllerScaleUpInterval := 55 * time.Minute
+
+	BeforeEach(func() {
+		By("Creating a test reconciler")
+		testReconciler = &SpannerAutoscalerReconciler{
+			scaleUpInterval: controllerScaleUpInterval,
+			clock:           testingclock.NewFakeClock(fakeTime),
+			log:             logr.Discard(),
+		}
+	})
+
+	It("should get controller default scaleup interval", func() {
+		want := controllerScaleUpInterval
+		sa := &spannerv1beta1.SpannerAutoscaler{
+			Status: spannerv1beta1.SpannerAutoscalerStatus{
+				LastScaleTime:          metav1.Time{Time: fakeTime.Add(-time.Minute)},
+				CurrentProcessingUnits: 2000,
+				DesiredProcessingUnits: 1000,
```

Comment:
For scaleup scenario, desiredPU should grater than currentPU?
```suggestion
				CurrentProcessingUnits: 1000,
				DesiredProcessingUnits: 2000,
```